pkg/index/job/save/usecase/save.go

```diff
@@ -0,0 +1,213 @@
+// Copyright (C) 2019-2023 vdaas.org vald team <vald@vdaas.org>
```

Comment:
`saved.go` is better if we should match file naming conventions.

Reply:
This is not a daemon but a job so I think `save.go` is ok for now. We should discuss the naming for all the jobs including this one later when we refactor.

Reply:
@vankichi @ykadowak 
Thank you for your comment!
> saved.go is better if we should match file naming conventions.

Yes, unlike other components, this component does not behave like daemon process. So it has no `d` prefix. 
I think that there is some discussion point about naming, so I will refactor it after all the jobs are implemented. 👍